qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -159,9 +160,17 @@ def run(self,
                 DeprecationWarning,
                 stacklevel=3)
 
+        # Add default OpenMP options
+        gpu = backend_options is not None and 'gpu' in backend_options.get('method', '')
```

Comment:
Backend options is a deprecated argument. Typically the method will already be in the qobj config